view/script/trial.js

```diff
@@ -17,6 +17,20 @@
         }]
     };
     var ctx = document.getElementById('trialChart').getContext('2d');
+    var differenceValue = 0;
+
+    document.getElementById('toDate').setAttribute('min', document.getElementById('fromDate').value);
+    $('#fromDate').on('change', function changetoDateValue () {
+        var fromDateValue = document.getElementById('fromDate').value;
+        var newtoDateValue = moment(fromDateValue).add(differenceValue, 'days').format('YYYY-MM-DD');
+
+        document.getElementById('toDate').value = newtoDateValue;
+    }).on('focus', function xyz (event) {
```

Comment:
this function is used to store previous date duration.
so `storeDateDiff` or `storeDuration` could be better names than `xyz`?